docs/custom-music/index.md

```diff
@@ -132,3 +132,27 @@ Once done, save and repackage your mod, and you'll find your audio replacement p
 | AMB_MainMenu_Loop      | 259073131  | Played in the background of the Main Menu         |
 | PD3_MainTheme_Option02 | 1028929042 | Identical to 1033817546, but not used by the game |
 | PD3_MainTheme_Option02 | 1033817546 | Main menu theme used in game                      |
+
+### Correctly looping/making a menu music mod with a length superior to the original's
+
+The way your 1033817546.ubulk is going to be handled in-game depends on 
+values contained within a .uexp file located in
+
+`PAYDAY3\Content\WwiseAudio\Events\Music\Menus` called `MusicMenu_MainMenu`
+
+Editing the values inside the uexp using a hex editor like HxD lets you set your own duration, loop times and fade in/fade out times.
+
+
+| Offset   | Value Type       | Value Scale  | Function               | Notes
+|:---------|:-----------------|:-------------|:-----------------------|:-------------------------------------------------------------------------------------------------------|
+| 0x146    | float64 (Double) | milliseconds | Music Duration         |    																									   |
```

Comment:
These are fixed offsets that only apply to the main menu track? I'd much rather have instructions that allow people to find these offsets for any track.

Reply:
An example on the main menu track wouldn't go amiss. For example, the main menu song is 192593 miliseconds long according to wwise. Just giving an example on that for the default uexp values should at least point complete hex editing newbies in the correct direction, maybe? I've passing knowledge about hex editing but this one just eludes me.

![image](https://github.com/MoolahModding/site/assets/86528533/03aceb5a-9fe7-4814-9dca-29cdbc6cd24c)
Apparently this part got removed at one point, which again helps at least pointing people at the solution I guess.